### PR TITLE
add ipcalc and sipcalc

### PR DESCRIPTION
**Pull request by mbakke**
Two simple IP calculators. Useful every blue moon.

### CODE REVIEW

File: pkgs/tools/networking/sipcalc/default.nix [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  name = "sipcalc-${version}";
+  version = "1.1.6";
+  src = fetchurl {
+    url = "http://www.routemeister.net/projects/sipcalc/files/${name}.tar.gz";
+    sha256 = "cfd476c667f7a119e49eb5fe8adcfb9d2339bc2e0d4d01a1d64b7c229be56357";
+  };
+  meta = {
+    description = "Advanced console ip subnet calculator";
+    homepage = "http://www.routemeister.net/projects/sipcalc/";
+    license = stdenv.lib.licenses.bsd3;
```
Comment (domenkozar) [inline pkgs/tools/networking/sipcalc/default.nix]: please set `meta.platforms`

File: pkgs/tools/networking/ipcalc/default.nix [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  };
+  buildInputs = [perl];
+  installPhase = ''
+    mkdir -p $out/bin
+    cp ipcalc $out/bin
+  '';
+  meta = {
+    description = "Simple IP network calculator";
+    homepage = "http://jodies.de/ipcalc";
+    license = stdenv.lib.licenses.gpl2Plus;
```
Comment (domenkozar) [inline pkgs/tools/networking/ipcalc/default.nix]: please set `meta.platforms`

**Comment by mbakke** [comment]
Added and fixed up.

**Comment by lethalman** [comment]
Travis passed, thanks.

**Pull request merged**